src/tools/rustfmt/Cargo.toml

```diff
@@ -36,11 +36,11 @@ generic-simd = ["bytecount/generic-simd"]
 annotate-snippets = { version = "0.9", features = ["color"] }
 anyhow = "1.0"
 bytecount = "0.6.4"
-cargo_metadata = "0.15.4"
+cargo_metadata = "0.18"
 clap = { version = "4.4.2", features = ["derive"] }
 clap-cargo = "0.12.0"
 diff = "0.1"
-dirs = "4.0"
+dirs = "5.0"
```

Comment:
Please revert the rustfmt changes and open a PR directly to rustfmt for the dependency changes. I'd like a chance to review the new dependencies. We had issues in the past when updating dirs (https://github.com/rust-lang/rustfmt/pull/5472)